docker-for-mac/index.md

```diff
@@ -331,14 +334,112 @@ This option removes/resets all Docker data _without_ a reset to factory defaults
 
 ![Uninstall or reset Docker Edge features](images/settings-uninstall-edge.png)
 
+## Adding TLS certificates
+
+You can add server side (registry) and client side certificates to verify the identity of these entities.
```

Comment:
@cyli makes sense, I'll change that now. I made some more updates, I think it's looking pretty good.